tests/members/test_members_services.py

```diff
@@ -68,6 +69,33 @@ def test_add_denied(
         data
     )
 
+@pytest.mark.parametrize("actor,role", [
```

Comment:
This is good, and very comprehensive 💪. Can we add another test, with just one use case:
- Having the flag enabled (with the fixture mentioned below)
- Adding a user (not group), to test that the permissions work properly (i.e. that they dont deny by default).

Reply:
The answer to this comment was posted internally on mattermost due to it's length.

Reply:
Can you add the answer here. If someone else has the same doubt they cannot see the mattermost chat ;)